wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -59,8 +59,9 @@
 @register.inclusion_tag("wagtailadmin/shared/breadcrumb.html", takes_context=True)
 def explorer_breadcrumb(
```

Comment:
happy if you want to rename this to `breadcrumbs` across the whole admin as part of the same PR that cleans up the move breadcrumbs that are no longer used.